src/lamplib/src/genny/tasks/auto_tasks.py

```diff
@@ -461,6 +463,11 @@ def write(self, tasks: List[GeneratedTask], write: bool = True) -> Configuration
                     f"{'Succeeded' if success else 'Failed'} to write to {output_file} from cwd={os.getcwd()}."
                     f"{raised if raised else ''}"
                 )
+                if self.op.execution != 0:
+                    SLOG.warning(
```

Comment:
Thoughts on making this an explicit error?

Reply:
I'd argue not an explicit error so that if someone restarts a whole patch, they don't get confusing errors here. I think that's the evergreen rationale for not errorring on it too.

Reply:
That's a great point!